libs/cardano-ledger-core/bench/Addr.hs

```diff
@@ -19,7 +19,7 @@ import Test.QuickCheck.Random (QCGen, mkQCGen)
 main :: IO ()
 main = do
   defaultMain $
-    map (\c -> env (generateAddrAsBytestring qcGen c) decodeAddrBench) (map (* 500) [1 .. 2])
```

Comment:
I can't figure out how this change impacts the result -  I will ask you on Wednesday!

Reply:
It does not affect it in any way. :smile: It is just clearer on what is going on.